### PR TITLE
Datetime picker filter

### PR DESCRIPTION
**Pull request by ldlsegovia**
Adds date time picker filter

```ruby
filter :created_at, as: :date_time_picker_filter
```

image

### CODE REVIEW

File: spec/features/inputs/date_time_picker_filter_input_spec.rb [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  end
+
+  context "setting max value only" do
+    before do
+      find("#q_created_at_lteq").set("1984-06-04 12:00")
+      click_filter_btn
+    end
+
+    it "shows filtered rows", js: true do
+      expect(page).to_not have_css("#invoice_5")
```
Comment (monkeyci) [inline spec/features/inputs/date_time_picker_filter_input_spec.rb:65]: Prefer not_to over to_not.

File: spec/features/inputs/date_time_picker_filter_input_spec.rb [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    it "shows filtered rows", js: true do
+      expect(page).to have_css("#invoice_5", count: 1)
+      expect(page).to have_css("#invoice_4", count: 1)
+      expect(page).not_to have_css("#invoice_3")
+      expect(page).not_to have_css("#invoice_2")
+      expect(page).not_to have_css("#invoice_1")
+    end
+  end
+
+  context "setting max value only" do
```
Comment (monkeyci) [inline spec/features/inputs/date_time_picker_filter_input_spec.rb:58]: Start context description with 'when', 'with', or 'without'.

File: spec/features/inputs/date_time_picker_filter_input_spec.rb [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    it "shows filtered rows", js: true do
+      expect(page).to_not have_css("#invoice_5")
+      expect(page).to have_css("#invoice_4", count: 1)
+      expect(page).to have_css("#invoice_3", count: 1)
+      expect(page).to have_css("#invoice_2", count: 1)
+      expect(page).to_not have_css("#invoice_1")
+    end
+  end
+
+  context "setting min value only" do
```
Comment (monkeyci) [inline spec/features/inputs/date_time_picker_filter_input_spec.rb:43]: Start context description with 'when', 'with', or 'without'.

File: spec/features/inputs/date_time_picker_filter_input_spec.rb [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+      find("#q_created_at_lteq").set("1984-06-04 12:00")
+      click_filter_btn
+    end
+
+    it "shows filtered rows", js: true do
+      expect(page).to_not have_css("#invoice_5")
+      expect(page).to have_css("#invoice_4", count: 1)
+      expect(page).to have_css("#invoice_3", count: 1)
+      expect(page).to have_css("#invoice_2", count: 1)
+      expect(page).to_not have_css("#invoice_1")
```
Comment (monkeyci) [inline spec/features/inputs/date_time_picker_filter_input_spec.rb:39]: Prefer not_to over to_not.

File: spec/features/inputs/date_time_picker_filter_input_spec.rb [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+  context "setting min and max values" do
+    before do
+      find("#q_created_at_gteq").set("1984-06-04 10:00")
+      find("#q_created_at_lteq").set("1984-06-04 12:00")
+      click_filter_btn
+    end
+
+    it "shows filtered rows", js: true do
+      expect(page).to_not have_css("#invoice_5")
```
Comment (monkeyci) [inline spec/features/inputs/date_time_picker_filter_input_spec.rb:35]: Prefer not_to over to_not.

File: spec/features/inputs/date_time_picker_filter_input_spec.rb [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+    visit admin_invoices_path
+  end
+
+  it "shows date time pickers", js: true do
+    expect(page).to have_css(".date-time-picker-input", count: 2)
+    expect(page).to have_css("#q_created_at_gteq", count: 1)
+    expect(page).to have_css("#q_created_at_lteq", count: 1)
+  end
+
+  context "setting min and max values" do
```
Comment (monkeyci) [inline spec/features/inputs/date_time_picker_filter_input_spec.rb:27]: Start context description with 'when', 'with', or 'without'.

**Review by gmq — APPROVED** [review]
LGTM 👍

**Pull request merged**